.github/workflows/codeql-analysis.yml

```diff
@@ -66,5 +66,8 @@ jobs:
     #   make bootstrap
     #   make release
 
+    - run: |
```

Comment:
We could also setup some cache to speed it up, see https://github.com/tektoncd/pipeline/pull/7728/files#diff-63bd641104d10e25f141d518a16b22a151d125e12701df2f9e79734b23b90188R64.

Reply:
done, updated